src/hooks/use-main-query.ts

```diff
@@ -9,20 +9,14 @@ const useMainQuery = () => {
   const { isAuthenticated } = useAuthContext()
   const { defaultLanguage } = useAppState()
 
-  let mainHeaders = {}
-  if (!isAuthenticated) {
-    mainHeaders = { credentials: "omit" }
-  }
-
-  const { data, refetch } = useQuery.main({
-    variables: { isAuthenticated, recentTransactions: 5 },
-    context: {
-      headers: { ...mainHeaders },
-    },
-    onCompleted: (completed) => {
-      setLocale(completed?.me?.language ?? defaultLanguage)
-    },
-  })
+  const { data, refetch } = isAuthenticated
```

Comment:
cant we just do an early return?

Reply:
i tried that first but the use-main-query hook needs the full type. this is patch to get us by until we remove the `galoy-client` dependency